src/node_errors.cc

```diff
@@ -140,8 +140,43 @@ static std::string GetErrorSource(Isolate* isolate,
   return std::string(buf);
 }
 
+void PrintStackTrace(Isolate* isolate) {
+  Local<Context> context = isolate->GetCurrentContext();
+  Local<String> error_string =
+      String::NewFromOneByte(isolate,
+                             reinterpret_cast<const uint8_t*>("Error"),
+                             v8::NewStringType::kInternalized,
+                             5)
+          .ToLocalChecked();
+  Local<String> stack_trace_limit_string =
+      String::NewFromOneByte(
+          isolate,
+          reinterpret_cast<const uint8_t*>("stackTraceLimit"),
+          v8::NewStringType::kInternalized,
+          15)
+          .ToLocalChecked();
+  MaybeLocal<Value> stack_trace_limit_val =
+      context->Global()
+          ->Get(context, error_string)
+          .ToLocalChecked()
+          .As<Object>()
+          ->Get(context, stack_trace_limit_string);
+  int frame_limit = 10;
+  if (!stack_trace_limit_val.IsEmpty()) {
+    Local<Value> stack_trace_limit_lval =
+        stack_trace_limit_val.ToLocalChecked();
+    if (stack_trace_limit_lval->IsNumber()) {
+      frame_limit =
+          stack_trace_limit_lval.As<Number>()->Int32Value(context).ToChecked();
```

Comment:
This `Int32Value(context)` should check for it's value either before with `IsInt32` or after with `FromMaybe(10)`.
Though I'd prefer getting double `->Value()` and then converting to uint32.

Also, I'm not sure if this accounts for `Error.stackTraceLimit = Infinity`.